### ISSUE TITLE
Add dark to https://hstspreload.org/

### ISSUE THREAD

**Issue by StachuDotNet** [darklang/dark#3829]
https://hstspreload.org

Currently blocked by #3745.
Also, we don't need to include the header for http calls - they only matter for https pages.

**Comment by pbiggar** [comment]
Why is it blocked by #3745? Is it a requirement to have correct headers before we can join?

**Comment by StachuDotNet** [comment]
merged into #5246
